library/game_state.py

```diff
@@ -155,9 +198,25 @@ def confirm_good_move(self, board: Board) -> bool:
 
         return self.good_move
 
+    def confirm_good_subgrid(self, board: Board) -> bool:
+        """Handle the entry of a bad subgrid choice"""
+        # TODO need to finish this. maybe move to game logic?
+        # guilty until proven innocent
+        self.good_move = False
+
+        working_subgrid = board.collect_subgrid(str(self.user_select_subgrid))
+        if board.check_grid_victory(working_subgrid) not in ("X", "O"):
+            self.good_move = True
+
+        return self.good_move
+
     def change_player(self) -> None:
         """Change Player"""
         if self.player_active == 1:
             self.player_active = 2
         else:
             self.player_active = 1
+
+    def game_over(self, term: blessed.Terminal) -> None:
+        """Placeholder game over function"""
+        print("Game over!")

```

Comment:
I think a better placeholder would just do `user_term[1] = "Game over!"` and then redrew the user terminal

Reply:
Changed this placeholder to use the info_term